velox/exec/tests/AggregationTest.cpp

```diff
@@ -1430,6 +1430,20 @@ TEST_F(AggregationTest, groupingSetsEmptyInput) {
       plan,
       "SELECT count(c2) as count_c2 FROM tmp WHERE c1 < 0 GROUP BY GROUPING SETS ((c1), ())");
 
+  // Distinct aggregations with GROUPING SETS.
```

Comment:
Shouldn't we clarify that this is over empty input?

Reply:
The entire test (named groupingSetsEmptyInput) is for empty inputs so its implicit.